src/main/openapi/ds-storage-openapi_v1.yaml

```diff
@@ -817,5 +862,29 @@ components:
         childrenIds: "null"
         parent: "null"
         children: "null"
-
-
+        
+        
+  # TODO: Inject this from properties when DRA-139 has been implemented        
+  securitySchemes:
+
+    KBOAuth:
+      type: oauth2
+      description: 'This API uses OAuth 2 with the implicit grant flow. [More info](https://api.example.com/docs/auth)'
+      flows:
+        implicit:   # <---- OAuth flow(authorizationCode, implicit, password or clientCredentials)
+          authorizationUrl: 'https://keycloak-devel-01.kb.dk/realms/ds-poc/protocol/openid-connect/auth'
+          #authorizationUrl: https://api.example.com/oauth2/authorize # Overridden by the configuration
```

Comment:
As in present and datahandler, please inject this with ${config.yaml.path}

Reply:
fixed